presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -332,7 +332,7 @@ private static Deserializer createDeserializer(Class<? extends Deserializer> cla
     private static void initializeDeserializer(Deserializer deserializer, Properties schema)
     {
         try {
-            deserializer.initialize(null, schema);
+            deserializer.initialize(new Configuration(), schema);
```

Comment:
I am changing this to use `new Configuration(false)` as loading the defaults involves XML parsing which is crazy expensive. See `HiveHdfsConfiguration` for how we avoid this by only loading defaults once and manually copying them out.